posthog/tasks/status_report.py

```diff
@@ -49,10 +49,20 @@ def status_report(*, dry_run: bool = False) -> Dict[str, Any]:
         (plugin_config.plugin.name for plugin_config in plugin_configs if plugin_config.enabled)
     )
 
-    for team in Team.objects.all():
+    org_usage_summary: Dict[str, Any] = {
```

Comment:
Nit: It's `Dict[str, int]`

It's imo important not to mess up terminology. We have 3 layers of abstraction:

instance > organization > team.

An instance may have multiple orgs, so this should be named `instance_usage_summary`

Reply:
Thank you for clarifying the layers!